internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -317,23 +357,22 @@ func resourceCosmosDbAccount() *pluginsdk.Resource {
 				Type:     pluginsdk.TypeSet,
 				Optional: true,
 				Computed: true,
-				ForceNew: true,
```

Comment:
Yes, but because in some cases `Capabilities` could actually be updated. When a user only wants to enable feature for an existing database, the database must be recreated. This is obviously not friendly enough. So I removed `ForeceNew` for the `Capabilities` in this PR, added the `Capabilities` update logic. The table below is updatable and the removal rules are confirmed by the service team.

> 
![image](https://user-images.githubusercontent.com/39109137/192260074-d529769f-e0a3-445e-b071-ce3f3aa00141.png)